test/test_spark_torch.py

```diff
@@ -343,14 +343,16 @@ def test_torch_direct_parquet_train(self):
                         store=store,
                         model=model,
                         optimizer=optimizer,
-                        loss=loss,
                         input_shapes=[[2]],
                         feature_cols=['features'],
                         label_cols=['y'],
                         batch_size=1,
                         epochs=3,
                         verbose=2)
 
+                    # To make sure that setLoss works with non-list loss.
+                    est.setLoss(loss)
+
```

Comment:
Can you add tests for `_calculate_loss_fn` in `remote.py`?  I think this problem could have been caught before if we had testing for multi-label scenarios.

Reply:
added.